lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py

```diff
@@ -92,12 +92,14 @@ def main(repo_csv, app_csv, dep_csv):
     Final output only includes paths which might contain views.
 
     """
+    # Maps theme names to a list of code owners (theme+squad), and squads to code owners (theme+squad)
```

Comment:
```suggestion
    # Maps theme name to a list of code owners in the theme, and squad to full code owner name.
    # Code owner is a string combining theme and squad information.
```

Suggested rewording to clarify purpose of maps.